### PR TITLE
Add formatting team

### PR DESCRIPTION
**Pull request by infinisil**
As [established by RFC 166](https://github.com/NixOS/rfcs/blob/master/rfcs/0166-nix-formatting.md#nix-formatter-team), see also https://github.com/NixOS/nixfmt/issues/153.

Before merging I'd like all members to give their okay with their entry. Ping @NixOS/nix-formatting

### CODE REVIEW

**Comment by Sereja313** [comment]
Could you please add my full name - Sergey Gulin?

**Comment by Sereja313** [comment]
Thank you!

File: src/content/teams/12_formatting.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  github: infinisil
+  title: Administrative leader, sponsored by Tweag and Antithesis
+- name: piegames
+  github: piegamesde
+  title: Main developer
+- name: Thomas Bereknyei
+  github: tomberek
+  title:
+- name: Joachim Ernst
+  github: 0x4A6F
```
[the remark this answers is not in the document]
Comment (infinisil) [reply src/content/teams/12_formatting.mdx]: Gotta love YAML!

**Review by 0x4A6F — APPROVED** [review]

**Review by dasJ — APPROVED** [review]

**Review by thilobillerbeck — APPROVED** [review]

**Comment by infinisil** [comment]
@piegamesde @tomberek Please also review

File: src/content/teams/12_formatting.mdx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+name: Nix Formatting Team
+description: Maintains the official Nix formatter and standard format.
+members:
+- name: Silvan Mosberger
+  github: infinisil
+  title: Administrative leader, sponsored by Tweag and Antithesis
```
Comment (mweinelt) [inline src/content/teams/12_formatting.mdx:7]: Is there a precedent for such a sponsors remark?

Comment (infinisil) [reply src/content/teams/12_formatting.mdx:7]: Me and @fricklerhandwerk also have that for the [docs team](https://nixos.org/community/teams/documentation/), as does @ConnorBaker from the [cuda team](https://nixos.org/community/teams/cuda/), all from Tweag

File: src/content/teams/12_formatting.mdx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- name: piegames
+  github: piegamesde
+  title: Main developer
+- name: Thomas Bereknyei
+  github: tomberek
+  title:
```
Comment (tomberek) [inline src/content/teams/12_formatting.mdx:13]:
```suggestion
  title: Supporting developer, sponsored by Flox
```

Comment (tomberek) [reply src/content/teams/12_formatting.mdx:13]: Non-blocking

**Review by tomberek — APPROVED** [review]
Looks fairly straight forward.

**Review by piegamesde — APPROVED** [review]

File: src/content/teams/12_formatting.mdx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+- name: "0x4A6F"
+  github: "0x4A6F"
+  title:
+- name: Sergey Gulin
+  github: Sereja313
+  title: from Serokell
```
Comment (Sereja313) [inline src/content/teams/12_formatting.mdx:19]:
```suggestion
  title: Sponsored by Serokell
```
Could you please also change this to "Sponsored by Serokell" to make it look more consistent with the others?

**Pull request merged**